diff --git a/PKGBUILD b/PKGBUILD
index 5b4e772..d5a61c9 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: William Tang <ttc0419@outlook.com>

 pkgname=unreal-engine-bin
-pkgver=5.7.0
+pkgver=5.7.2
 pkgrel=1
 pkgdesc="The world's most open and advanced real-time 3D creation tool"
 arch=('x86_64')
@@ -16,7 +16,7 @@ optdepends=('android-ndk: Android build support'

 DLAGENTS+=('manual::/usr/bin/echo \ \ Note: Please download the zip file manually from https://www.unrealengine.com/linux')
 source=("manual://Linux_Unreal_Engine_${pkgver}.zip" 'unreal-engine.desktop' 'unreal-engine.xml' {16,24,32,48,64,256}.png)
-sha256sums=('3a308efc60d8852ce289b44ce5fddf77cfa2012b73ae728e0b9a970791087ce3'
+sha256sums=('142d18594087d7377952a038cafce628d22be9eab670b706ebfe20cfd443b3cc'
             '8a71e10dee1d44ad6f85d993bdf25dd4835f6415e983125ac79a912ee0c3d905'
             '752dc3628639adb84ebda281040ec106227431eb7a5f92375511d96c6ceacf91'
             '16197bc88f2ec77ad2690fae280bf3584c54c7d7b018d8fa4086b7c00fd4effc'
